examples/supertuxkart/Makefile

```diff
@@ -47,3 +47,7 @@ test:
 # check if hosted on Google Cloud Registry
 gcr-check:
 	gcloud container images describe $(image_tag)
+
+#output the tag for this image
+echo-image-tag:
+	@echo $(image_tag)
```

Comment:
Is that ok to make a code change with release?

Reply:
I usually find it's a good time to improve on automation for the release process :shrug: allows me to remove toil at the time I'm incentivised to do it.

I figure as long as it's limited to release aspects, it should be okay :thinking: